AMW_angular/io/src/app/servers/servers-filter/servers-filter.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="container">
+    <div class="card">
```

Comment:
Instead of returning a `card` the filter component should only return a container. similar to apps-filter.component.html
The card should be used for both components in server -> #1032

Reply:
This should be resolved with #1032